src/components/registerPlugins.ts

```diff
@@ -87,7 +88,8 @@ export default async function registerPlugins() {
         plugin.description !== dbPlugin.description ||
         plugin.path !== dbPlugin.path ||
         plugin.function_name !== dbPlugin.function_name ||
-        !deepEquality(plugin.required_parameters, dbPlugin.required_parameters))
+        !deepEquality(plugin.required_parameters, dbPlugin.required_parameters) ||
+        !deepEquality(plugin.output_properties, dbPlugin.output_properties, true))
```

Comment:
Can this checks be moved to a utility function? I don't understand what this is checking. So I thought it would be easier to see if this was just a call a simple utility function (in the same file) so the code here looks clearar and the name of this function already says what this is doing.

Reply:
It's checking to see if the plugin details its just scanned from the folders have changed compared with what's in the database (then updates the database if so).

Moved comparison to own function at the bottom.

Reply:
Thanks!